srcpkgs/helio-sequencer/template

```diff
@@ -0,0 +1,48 @@
+# Template file for 'helio-sequencer'
+pkgname=helio-sequencer
+version=3.12
+revision=1
+_gitrev="61f7db380f083b424418d65f4e26addc1370ddea"
+archs="i686 x86_64"
+hostmakedepends="git pkg-config"
+makedepends="freetype-devel libX11-devel libXinerama-devel libXrandr-devel libXcursor-devel libXcomposite-devel MesaLib-devel libfreeglut-devel libcurl-devel alsa-lib-devel jack-devel libsigc++-devel"
+depends="curl freetype libXinerama libglvnd alsa-lib"
```

Comment:
Shared-library dependencies should be automatically detected. Use this field for dependencies which aren't automatically picked up, and only *hard* dependencies--those without which the application will not function.

Reply:
As far as i know helio-sequencer will not make sound without alsa-lib, it will not render without libglvnd etc. So it needs these depends i think

Reply:
I have a suspicion these will be automatically picked up. Try it and see.